octoprint_mrbeam/static/css/svgtogcode.css

```diff
@@ -588,7 +588,6 @@ li.material_entry.custom.incompatible_material::after {
   content: "";
   display: block;
   height: 0.5em;
-  background-color: white;
   position: relative;
```

Comment:
Better change the 'background' CSS into 'background-color' as well

Reply:
This need to be background as it will not work as background-color